doc/string/new.rdoc

```diff
@@ -1,34 +1,41 @@
-Returns a new \String that is a copy of +string+.
+Returns a new \String object containing the given +string+.
 
-With no arguments, returns the empty string with the Encoding <tt>ASCII-8BIT</tt>:
+The +options+ are optional keyword options (see below).
 
-  s = String.new
-  s # => ""
-  s.encoding # => #<Encoding:ASCII-8BIT>
+With no argument given and keyword +encoding+ also not given,
+returns an empty string with the Encoding <tt>ASCII-8BIT</tt>:
 
-With optional argument +string+ and no keyword arguments,
-returns a copy of +string+ with the same encoding:
+  s = String.new # => ""
+  s.encoding     # => #<Encoding:ASCII-8BIT>
 
-  String.new('foo')               # => "foo"
-  String.new('тест')              # => "тест"
-  String.new('こんにちは')          # => "こんにちは"
+With argument +string+ given and keyword option +encoding+ not given,
+returns a new string with the same encoding as +string+:
+
+  s0 = 'foo'.encode(Encoding::UTF_16)
+  s1 = String.new(s0)
+  s1.encoding # => #<Encoding:UTF-16 (dummy)>
+  s0 = ''.encode(Encoding::UTF_16)
+  s1 = String.new(s0)
+  s1.encoding # => #<Encoding:UTF-16 (dummy)>
```

Comment:
These two examples seems to be showing the exact same thing? What's the second example showing?